worldengine/draw.py

```diff
@@ -161,14 +161,47 @@ def __setitem__(self, item, value):
         self.pixels[item] = value
 
 
-def draw_simple_elevation(data, width, height, sea_level, target):
+def draw_simple_elevation(world, sea_level, target):
     """ This function can be used on a generic canvas (either an image to save
         on disk or a canvas part of a GUI)
     """
-    for y in range(height):
-        for x in range(width):
-            e = data[y][x]#TODO: numpy-notation [y,x] is a little faster; all passed arrays would have to be numpy-arrays
-            r, g, b = elevation_color(e, sea_level)
+    min_elev_sea = None
+    max_elev_sea = None
+    min_elev_land = None
+    max_elev_land = None
+    for y in range(world.height):
```

Comment:
This replaces lines 170 till 191, inclusive.
mask is a masked array, every part of the map that corresponds to ocean is masked out. Therefore calling ```mask.min()``` etc. will give you the minimum of the non-masked elements.

Then the mask is inverted (now everything that is land is masked out) and the functionality of ```mask.min()``` changes accordingly.